daemon/algod/api/client/restClient.go

```diff
@@ -220,7 +220,13 @@ func (client RestClient) submitForm(response interface{}, path string, request i
 
 	if decodeJSON {
 		dec := json.NewDecoder(resp.Body)
-		return dec.Decode(&response)
+		decodeErr := dec.Decode(&response)
+		switch decodeErr {
+		case io.EOF:
+			return nil
+		default:
+			return decodeErr
+		}
```

Comment:
For most requests, this would be a real error and this logic would incorrectly ignore the error.

Could you find a way to skip decoding for the delete command?

Reply:
Could you find a general solution so that the next person adding a `DELETE` endpoint doesn't run into the same issue?